src/util/validatePeer.ts

```diff
@@ -1,14 +1,22 @@
 const url = require('url')
 const IGNORE_LIST = ['localhost', '0.0.0.0', 'local.codius.org']
+import * as reduct from 'reduct'
+import Config from '../services/Config'
+const config = reduct()(Config)
 
 export function shallowValidatePeer (peer?: string): boolean {
   if (!peer) {
     return false
   }
+
+  if (!config.devMode && !peer.includes('https')) {
```

Comment:
this should parse the url and check the protocol. Otherwise `nameusinghttps.com` will be filtered out.

Reply:
If devmode is true, we should return true no matter what